src/app/features/product-page/containers/product-page/product-page.component.html

```diff
@@ -30,37 +30,29 @@
     (clicked)="removeProductFromWishlist()"
   ></ile-button>
   <div class="ratings" *ngIf="product?.owner.id !== loggedInUser?.id">
-    <fieldset
-      class="rating"
-      [ngStyle]="{ 'pointer-events': isRated ? 'none' : 'auto' }"
-    >
-      <input
-        *ngFor="let value of [5, 4.5, 4, 3.5, 3, 2.5, 2, 1.5, 1, 0.5]"
-        type="radio"
-        name="rating"
-        id="star-{{ value }}"
-        (input)="updateProductRating(value)"
-      />
-      <label
-        *ngFor="
-          let value of [5, 4.5, 4, 3.5, 3, 2.5, 2, 1.5, 1, 0.5];
-          let i = index
-        "
-        for="star-{{ value }}"
-        [ngClass]="{ full: i % 2 === 0, half: i % 2 !== 0 }"
-      ></label>
+    <fieldset class="rating" [ngStyle]="{'pointer-events': isRated ? 'none' : 'auto'}">
```

Comment:
Why did you gave up on the for loop?

Reply:
I thought that don't break the styles but it did.. So I just revert it..